index.src.html

```diff
@@ -3415,29 +3417,39 @@
   VideoFrame(data, init)
 </dfn>
 1. If |init| is not a [=valid VideoFrameBufferInit=], throw a {{TypeError}}.
-2. Let |defaultRect| be «[ "x:" → `0`, "y" → `0`,
+1. Let |defaultRect| be «[ "x:" → `0`, "y" → `0`,
     "width" → |init|.{{VideoFrameBufferInit/codedWidth}}, "height" →
         |init|.{{VideoFrameBufferInit/codedWidth}} ]».
-3. Let |overrideRect| be `undefined`.
-4. If |init|.{{VideoFrameBufferInit/visibleRect}} [=map/exists=], assign its
+1. Let |overrideRect| be `undefined`.
+1. If |init|.{{VideoFrameBufferInit/visibleRect}} [=map/exists=], assign its
     value to |overrideRect|.
-4. Let |parsedRect| be the result of running the [=VideoFrame/Parse Visible
+1. Let |parsedRect| be the result of running the [=VideoFrame/Parse Visible
         Rect=] algorithm with |defaultRect|, |overrideRect|,
         |init|.{{VideoFrameBufferInit/codedWidth}},
         |init|.{{VideoFrameBufferInit/codedHeight}}, and
         |init|.{{VideoFrameBufferInit/format}}.
-5. If |parsedRect| is an exception, return |parsedRect|.
-6. Let |optLayout| be `undefined`.
-7. If |options|.{{VideoFrameBufferInit/layout}} [=map/exists=], assign its value
+1. If |parsedRect| is an exception, return |parsedRect|.
+1. Let |optLayout| be `undefined`.
+1. If |options|.{{VideoFrameBufferInit/layout}} [=map/exists=], assign its value
     to |optLayout|.
-8. Let |combinedLayout| be the result of running the [=VideoFrame/Compute
+1. Let |combinedLayout| be the result of running the [=VideoFrame/Compute
     Layout and Allocation Size=] algorithm with |parsedRect|,
     |init|.{{VideoFrameBufferInit/format}}, and |optLayout|.
-9. If |combinedLayout| is an exception, throw |combinedLayout|.
-10. If `data.byteLength` is less than |combinedLayout|’s
+1. If |combinedLayout| is an exception, throw |combinedLayout|.
+1. If `data.byteLength` is less than |combinedLayout|’s
     [=combined buffer layout/allocationSize=], throw a {{TypeError}}.
-11. Let |resource| be a new [=media resource=] containing a copy of |data|. Use
-        {{VideoFrameBufferInit/visibleRect}} and {{VideoFrameBufferInit/layout}}
+1. For each |transferable| in |init|.{{VideoFrameBufferInit/transfer}}:
+    1. If {{platform object/[[Detached]]}} internal slot is `true`,
+        then throw a {{DataCloneError}} {{DOMException}}.
+1. If |init|.{{VideoFrameBufferInit/transfer}} contains an ArrayBuffer referenced
+    by |data|:
+    1. Let |resource| be a new [=media resource=] referencing pixels in |data|
+        without a copy.
+        Use {{VideoFrameBufferInit/visibleRect}} and {{VideoFrameBufferInit/layout}}
+        to determine where in |data| the pixels for each plane reside.
+1. Otherwise:
+    1. Let |resource| be a new [=media resource=] containing a copy of |data|.
+        Use {{VideoFrameBufferInit/visibleRect}} and {{VideoFrameBufferInit/layout}}
```

Comment:
This is however useful because it can lead to copying less data, but really the paragraph below is enough.

Reply:
This is a `non-transfer` case that we have in the spec now. It's a copy of the step 11.

Reply:
Dropped the wording.